Add Stampede2 scaling results for Skylake nodes

naliboff opened the pull request:
Results for the Skylake nodes on Stampede2. The KNL nodes shower fairly poor results early on, but I can try running a full set of scaling results on those as well. However, the Skylake nodes have significantly more memory per core available, so really not much motivation to use the KNL nodes.

gassmoeller commented:
Thanks for the extra data! I will not be able to look at the data for the next days, but the structure looks correct and I will merge this when the following question is resolved. There is one binary file (._output_4_2_1) that should likely not be part of the pull request. Is this by accident?

naliboff commented:
@gassmoeller - Yes, that file is definitely there by accident! I removed the file and updated the pull request. Another pull request coming in a minute with data from Blue Waters, but no rush to look at it!

gassmoeller commented:
Thanks!
